File: Coders/DSA Concepts/BubbleSort.java
public class BubbleSort {
    public static void bubbleSort(int[] array){
        for (int i = array.length-1; i >0; i--) {
            for (int j = 0; j < i; j++) {
                if (array[j]>array[j+1]){
                    int temp = array[j];
                    array[j]=array[j+1];
                    array[j+1]=temp;
                }
            }
        }
    }
}

File: Coders/DSA Concepts/BubbleSort.py
#Python Program For Bubble Sort
def BubbleSort(mylist):
    for i in range(len(mylist)-1,0,-1):
        for j in range(i):
            if mylist[j]>mylist[j+1]:
                temp = mylist[j]
                mylist[j] =mylist[j+1]
                mylist[j+1] = temp   
mylist=[3,1,5,0]
print("Before Sorting---> ",mylist)
BubbleSort(mylist)
print("After Sorting---> ",mylist)
